### PR TITLE
Corrections for symmetry factor feature in GRChombo

### PR DESCRIPTION
**Pull request by the-florist**
Corrected the params files and any uses of AMRReductions in both of the Examples, to account for the integral symmetry factor feature in GRChombo (see GRChombo Issue No. 197). Specifically:

1. Added the symmetry_correction flag to the params files (under the Boundary Condition parameters)
2. Provided the chombo parameter storing the symmetry factor to every instance of the norm and sum AMRReductions functions, in the specificPostTimeStep() function for each example.

Note that the symmetry factor is calculated entirely in the ChomboParameters.h file of the main GRChombo repo.

### CODE REVIEW

File: Examples/BoostedBHComplexScalar/params.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@
 verbosity = 0
 
 # location / naming of output files
-# output_path = "" # Main path for all files. Must exist!
+output_path = "/nfs/st01/hpc-gr-epss/eaf49/PR-dump" # Main path for all files. Must exist!
```
Comment (KAClough) [inline Examples/BoostedBHComplexScalar/params.txt:10]: I think your own path crept in here!

File: Examples/BoostedBHComplexScalar/params.txt [hunk 2]
```diff
@@ -123,6 +123,11 @@ vars_parity            = 0 0 0 0   #phi and Pi (Re and Im)
 vars_parity_diagnostic = 0 0 0     #chi, rhoLinMom and rhoEnergy
 		         0 0 0     #fluxLinMom, fluxEnergy and sourceLinMom
 			 
+# Additionally, if reflective boundaries selected, 
+# choose if symmetry factor will be applied to integrals.
+# 0 = false (default), 1 = true
```
Comment (KAClough) [inline Examples/BoostedBHComplexScalar/params.txt:128]: I would default to true, as I can't imagine people would not want to correct it.

File: Examples/KerrBHScalarField/params.txt [hunk 3]
```diff
@@ -119,7 +119,12 @@ lo_boundary = 1 1 2
 vars_parity            = 0 0 0 0   #phi and Pi (Re and Im)
 vars_parity_diagnostic = 0 0 0     #chi, rhoEnergy and rhoAngMom
 		         0 0 0     #sourceAngMom, fluxAngMom and fluxEnergy
-			 
+
+# Additionally, if reflective boundaries selected, 
+# choose if symmetry factor will be applied to integrals.
+# 0 = false (default), 1 = true
```
Comment (KAClough) [inline Examples/KerrBHScalarField/params.txt:125]: Ditto here, default to true and comment out the symmetry_correction =1 line as we do for other defaulted params.

**Review by KAClough — COMMENTED** [review]
- Could you also correct the symmetry factors in the python diagnostics scripts? I guess they are now not needed?

- One of the intel tests seems to fail - I am not great with these but maybe @SamuelBrady can help...

**Comment by KAClough** [comment]
Apologies, the new PR I accepted seems to have broken a few more of the tests, if you could pull this into here (rebase to it?) and fix those at the same time that would be amazing...

**Comment by dinatraykova** [comment]
> * Could you also correct the symmetry factors in the python diagnostics scripts (one in each example)? They should not now be needed.
> * One of the intel tests seems to fail - I am not great with these but maybe @SamuelBrady can help...

Changed the intel test as in GRChombo, works now

**Pull request merged**